Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN mkdir -p /go/src/github.com/asecurityteam/asset-inventory-api
 WORKDIR $GOPATH/src/github.com/asecurityteam/asset-inventory-api
 COPY --chown=sdcli:sdcli . .
 RUN sdcli go dep
-RUN GO111MODULE=on GOFLAGS='-mod=vendor' go get github.com/gobuffalo/packr/v2/packr2
+RUN GO111MODULE=on go get github.com/gobuffalo/packr/v2/packr2
 RUN packr2
 RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -o /opt/app main.go
```

Comment:
Task: I'm not sure if the GOFLAGS var is needed for the `go get` on line 6, but it should be passed here on line 8 so `go build` uses the vendored dependencies, rather than pulling directly from source.

Reply:
+1